src/App.js

```diff
@@ -9,8 +12,11 @@ function App() {
 
   return (
     <div className="App d-flex">
-      <Navigation />
       <Routes>
+        <Route path="/home" element={(<Navigation />)} />
```

Comment:
The navigation component should appear in all paths except the `Homepage` and the `Login and Register` paths.
This can be achieved with the help of the pathname.
Example
`import { useLocation } from 'react-router-dom';`
And inside your `App` function,
  ```
const location = useLocation();
  const { pathname } = location;
```
Then inside the `return` `div`,
```
{ (pathname.indexOf('sign') !== 1 && pathname.length !== 1) || (pathname.indexOf('sign') !== 1 && pathname.endsWith('/') === false)
  ? <Navigation />
  : ''}
```